source/manual/merge-pr.html.md

```diff
@@ -64,6 +66,14 @@ git push --set-upstream origin thomasjhughes-patch-1
 
 Change `thomasjhughes` to the GitHub username of the contributor and `patch-1` to the name of the branch in their fork.
 
+#### Dependabot
+
+Dependabot raises PRs whenever it sees new versions of gems available that are required by our applications.
+
+Dependabot is an external contributor and is therefore subject to the same due diligence checks set out above as any other external contributor would have to go through, requiring two people from GDS to approve the PR before it can be merged. Particular attention should be paid to the changelog(s) of the upgraded gem(s) to ensure that no unintended side-effects are introduced.
+
+The one exception to the number of GDS people required to review the PR is if the gems being upgraded consist of only GOV.UK-owned gems or gems in the `test` block of the Gemfile - see [Manage Ruby dependencies with Dependabot](/manual/manage-ruby-dependencies.html) for more information.
```

Comment:
So I've been merging Dependabot Pull Requests for GOV.UK gems, without waiting for a second review. But in many of these cases, Dependabot has also changed the Gemfile.lock to update other gems as well.

Seen as you're updating the documentation, it might be good to try and clarify what should be done with Pull Requests relating to internal gems, with external updates mixed in? As a strict reading of this paragraph, seems at least to disagree with what I and others are doing.

Reply:
I think in this case we would _have_ to follow the process of requiring two reviews, regardless of the fact that internal gems have also been upgraded, because of the presence of the external gems that have also been upgraded. I'm happy to elaborate this in the above if it isn't clear.

Reply:
Given that I think that differs from what I'm doing at the moment, I think it would be worth being very explicit.

Reply:
Updated to provide more clarity.

Reply:
Great, thanks :+1: